src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
Yeah I get that, and was thinking the same thing, but then I noticed that all the containing components had to keep track of the selected item anyway. When the parent receives the selection change event, it's storing that somewhere - it doesn't wait, and then ask the component what the current selection is at some future point to post off a request or something, which is what lead me to thinking about this.